parsl/data_provider/globus.py

```diff
@@ -15,6 +15,10 @@
 SCOPES = ('openid '
           'urn:globus:auth:scope:transfer.api.globus.org:all')
 
+token_path = os.path.join(os.path.expanduser('./'), '.parsl')
+if not os.path.isdir(token_path):
+    os.mkdir(token_path)
+TOKEN_FILE = os.path.join(token_path, '.globus.json')
```

Comment:
this looks like some partial reversion of the changes in 9fd8dcda5585259f5dcc04e19899ec324a8edb6e rather than to do with the PR topic?

Reply:
Ah, yep, this is the 'not useful' part of the hacking around at FNAL.

Reply:
is it the result of some problem you are having with the existing `master` globus code?

Reply:
no no, that was FNAL's fault not parsl's... this works at other sites.